### ISSUE TITLE
Week 3 day 4 ex2 Innacurate audit output

### ISSUE THREAD

**Issue by eslopfer** [01-edu/Branch-AI#37]
When removing the punctuation, it shouldn't remove the spaces as well. In the audit file it's removing the last space.

Right output would be: `Remove this from  the sentence  `, with 2 spaces after from.
Output in the audit is `Remove this from  the sentence`.

If extra spaces should be removed it should be specified in the description, and in that case the extra space after from should be removed as well. Either way what is asking on the audit file is not correct.

**Comment by brad-gh** [comment]
Good point, thanks. I'll add a comment to precise that spaces doesn't matter here because depending on the tool they use to remove the punctuation it could also remove double spaces.
